src/docs/product/performance/trace-view.mdx

```diff
@@ -0,0 +1,51 @@
+
+Currently, the trace view will only show up to 100 transactions from the trace. Additional transactions can be found through [Discover](/product/discover-queries).
+
+## Trace View
+
+The trace view provides a split view. On the left, you can view the trace's transaction tree. On the right, you can view a representation of each transaction as a colored rectangle. The first row lists the trace itself, with all other spans descending from that trace.
+
+You can display these views from:
+- [Transaction Summary](/product/performance/transaction-summary/): Select the [Performance Homepage](/product/performance/), the affected transaction, then click a trace ID.
+- [Event Details](/product/performance/event-detail/): Click the trace ID beneath the quick trace on the top right.
+- [Issue Details](/product/issues/issue-details/): Click "View Full Trace" above the quick trace.
+- [Query Builder](/product/discover-queries/query-builder/): Build a query with the trace column, then click a trace ID in the table.
+
+Trace IDs are linked to the corresponding Trace Detail.
+
+## Waterfall
+
+The waterfall view is a split view. On the left, you can view the trace's transaction tree. On the right, each transaction displays as a horizontal bar.
+
+## Transaction Details
+
+Clicking a transaction row expands the details of that transaction to view many of the attached properties, such as tags and other field data. With the transaction name, you can view the [Transaction Summary](/product/performance/transaction-summary/) of that transaction across traces. Click "View Summary" to see that summary. As well with the transaction ID, you can see the [event details](/product/performance/event-detail/) of that transaction by clicking on "View Transaction".
+
+# Troubleshooting
+
+## Orphan Traces and Broken Subtraces
+In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in the first service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, we link the transactions with a dashed line since they can no longer be directly connected to the root.
+
+Broken subtraces may be caused by:
+- [Ad blockers](/platforms/javascript/troubleshooting/#dealing-with-ad-blockers) may prevent transactions in browsers being sent, but HTTP requests to backend projects will still create child transactions
+- [Rate-limiting](/product/accounts/quotas/#limiting-events) on a project may cause only some events to be sent to Sentry
+- [Project permissions](/product/accounts/membership/#restricting-access) may mean you do not have access to transactions in another project
+
+## Multiple Roots
+
+Each trace ID should have only one transaction without any parents -- the root. Included instrumentation should prevent multiple roots; however, if the trace ID of your transactions are being set using custom instrumentation, you may encounter multiple roots.
```

Comment:
Something like this? Because there can be multiple transactions without any parents (multiple orphans case), but multiple roots happen when there are more than 1 top-level parent.

```suggestion
Each trace ID should have only one transaction with a top-level parent -- the root. Included instrumentation should prevent multiple roots; however, if the trace ID of your transactions is being set using custom instrumentation, you may encounter multiple roots.
```

Reply:
Hmm not sure if I understand what a `top-level parent` is in the context of transactions.

Reply:
Sorry, a top-level parent is just referring to the root (probably better since the docs explain what a root is).

Now that I re-read this, I think my confusion around the sentence is specifically around the grammar. Can we change it from

> Each trace ID should have only one transaction without any parents -- the root.

to something like

> Each trace ID should have only one root, a trace without any parents.